test/deprecated/classes/owned-release-method.good

```diff
@@ -0,0 +1 @@
+owned-release-method.chpl:7: warning: owned.release method is deprecated - please use owned.release type function instead
```

Comment:
I don't think I caught this above, but "type method" would be the better / more precise term to use here.